selfdrive/car/tests/test_car_interfaces.py

```diff
@@ -18,15 +18,16 @@
 from openpilot.selfdrive.controls.lib.latcontrol_torque import LatControlTorque
 from openpilot.selfdrive.controls.lib.longcontrol import LongControl
 from openpilot.selfdrive.test.fuzzy_generation import DrawType, FuzzyGenerator
+from functools import lru_cache
 
 ALL_ECUS = {ecu for ecus in FW_VERSIONS.values() for ecu in ecus.keys()}
 ALL_ECUS |= {ecu for config in FW_QUERY_CONFIGS.values() for ecu in config.extra_ecus}
 
 ALL_REQUESTS = {tuple(r.request) for config in FW_QUERY_CONFIGS.values() for r in config.requests}
 
-MAX_EXAMPLES = int(os.environ.get('MAX_EXAMPLES', '40'))
-
+MAX_EXAMPLES = int(os.environ.get('MAX_EXAMPLES', '10'))
 
+@lru_cache(maxsize=1024)
 def get_fuzzy_car_interface_args(draw: DrawType) -> dict:
```

Comment:
You don't want to cache the function that returns a new example, or you defeat the purpose of running this multiple times. Maybe building the strategy is slow, and you can cache that though. You'll want to measure that